client/ayon_harmony/api/plugin.py

```diff
@@ -263,3 +268,94 @@ def get_instance_attr_defs(self):
                 label="Render target"
             )
         ]
+
+
+class HarmonyAutoCreator(HarmonyCreatorBase, AutoCreator):
+
+    enabled = True
+
+    def create(self):
+
+        variant = None
+        if self.default_variants:
+            variant = self.default_variants[0]
```

Comment:
`variant` should never be `None` right?

Also, you can just call `self.default_variant` ;) if `default_variant` attribute is not set, it's created in `__init__` with a wrapper that will fall back to the first vasriant of `default_variants`?

Reply:
There is/was some confusion between `default_variant` and `default_variants` and I am not sure, what was result. `default_variants` seem more universal.
(Wasn't that studio might want to provide set of expected variants for artist, these will show up in Publisher, priority order is kept, so separate `default_variant` is superfluous?)

Reply:
Hm, but `Publisher` doesn't allow dropdown for variants, only `Traypublisher` does, pwned.